src/sonic-yang-models/yang-models/sonic-system-aaa.yang

```diff
@@ -43,8 +43,8 @@ module sonic-system-aaa {
                     default "local";
                 }
 
-                must 'not(./type = "authorization" and contains(./login, "tacacs+") and not(/tacacs:sonic-system-tacacs/tacacs:TACPLUS/tacacs:global/tacacs:passkey))' {
-                    error-message "Authorization with 'tacacs+' is not allowed when passkey not exists.";
+                must 'not(./type = "authentication" and contains(./login, "tacacs+") and not(/tacacs:sonic-system-tacacs/tacacs:TACPLUS/tacacs:global/tacacs:passkey))' {
+                    error-message "Authentication with 'tacacs+' is not allowed when passkey not exists.";
```

Comment:
Should we have 2 `must` or just one `must`?

Reply:
For our case, authentication is enough. It is a superset of authorization

Reply:
Let me be more clear, if
1. authentication is local
2. authorization is tacacs+
3. no passkey

Is it a valid config or not?

Reply:
It is not valid in Azure. Becuase Azure requires a passkey to connect to tacacs server.

Reply:
Offline discussed, the conclusion is that this is valid config.